packages/rum/src/types/record.ts

```diff
@@ -36,6 +39,16 @@ export interface IncrementalSnapshotRecord {
   type: typeof RecordType.IncrementalSnapshot
   timestamp: TimeStamp
   data: IncrementalData
+  recordId?: number
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I would just call this property `id`, since it is (more or less) a `Record` property I think it's explicit enough to understand that it is a record id